### DISCUSSION TITLE
Support for scrolling menu items

### DISCUSSION THREAD

**Discussion by thijstriemstra** [forntoh/LcdMenu#30] [Ideas]
Sometimes the length of the label for a menu item is longer than 16 chars, cutting of the label. Would be cool if these chars can scroll on that line.

**Comment by forntoh** [comment]
**Update:** ItemInput will support this and will be in the the release version `4.3.0`, thanks to @ShishkinDmitriy ❤️

**Comment by forntoh** [comment]
Added in #241 🚀
